packages/vue/src/components/organisms/SfCollectedProduct/SfCollectedProduct.js

```diff
@@ -62,14 +61,13 @@ export default {
   },
   watch: {
     qty(value) {
-      const qty = toInt(value);
+      const qty = parseInt(value);
```

Comment:
```suggestion
      const qty = parseInt(value, 10);
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

"An integer between 2 and 36 that represents the radix (the base in mathematical numeral systems) of the string. Be careful — **this does not default to 10.**"

Reply:
Good catch! Just read that in a JS book the other day. Although this isn't my code, I will fix it in this PR.